pyproject.toml

```diff
@@ -34,6 +34,9 @@ skip_glob = [
   "report"
 ]
 
+[tool.pylint]
+max-line-length = 120
```

Comment:
Where's this value coming from? 120 seems like a stretch

Reply:
Updated. 

I followed the length in the [snap template](https://git.launchpad.net/bootstack-charms-spec/tree/templates/snap/pyproject.toml.j2#n5). But I checked the discussion in team chat, since `charmcraft init` is using 99 by default, we should align pylint config with it.

Reply:
Can you please update this also in our templates. Thanks